### DISCUSSION TITLE
How to customize styling?

### DISCUSSION THREAD

**Discussion by m3c-ode** [wojtekmaj/react-date-picker#468] [Q&A]
Hi there,

I am unable to grasp how to do customer styling with a Next.JS app. I Understand NextJS can't load the default styling and we can copy the .css file, but not sure how to do it from there. Would someone be able to offer an example?
Thank you.

**Comment by wojtekmaj** [comment]
You would need to be a bit more specific. What is that you're trying to modify in CSS, but cannot? I get that internal classes are not very well documented, so I'll need more info to help out.

**Reply by m3c-ode** [reply]
Thanks for your answer!
I meant that once you import the "no-style" version because of NextJS, how do you do to actually import the file into your project, and start customizing the date-picker as you wish? Where do you put your CSS so it has an effect?

**Reply by wojtekmaj** [reply]
You can put the file wherever you like, or even import it directly from npm module. I'd just import it in the same place you import the date picker component :)

**Reply by m3c-ode** [reply]
I've been able to import it where I use the component. But I am having quite some issues editing the styling in my NextJS project.

**Reply by wojtekmaj** [reply]
Um... Okay? What do you expect me to do about it? I asked if there's something specific you're trying to achieve, you provided no information. I don't know what can I do to help you.
